account_payment_order/wizard/account_payment_line_create_view.xml

```diff
@@ -15,6 +15,7 @@
             <group name="main">
                 <field name="order_id" invisible="1"/>
                 <field name="date_type"/>
+                <field name="due_on"/>
                 <field name="move_date" attrs="{'required': [('date_type', '=', 'move')], 'invisible': [('date_type', '!=', 'move')]}"/>
                 <field name="due_date" attrs="{'required': [('date_type', '=', 'due')], 'invisible': [('date_type', '!=', 'due')]}"/>
```

Comment:
date_type, due_on and move_date/due_date should be on the same line

So it reads like
* Due Date is before or equal to <some date>
* Move Date is equal to <some date>